main/pipeline/pipeline.cc

```diff
@@ -958,50 +957,6 @@ void package(core::GlobalState &gs, absl::Span<ast::ParsedFile> what, const opti
 #endif
 }
 
-// packager intentionally runs outside of rewriter so that its output does not get cached.
```

Comment:
The GitHub diff here makes this look weird, but what I actually deleted is not `buildPackageDB`, but rather `pipeline::package`, which only had one user and it was actually a no op.